frame/bounties/src/lib.rs

```diff
@@ -435,7 +435,9 @@ decl_module! {
 								} else {
 									// Else this is the curator, willingly giving up their role.
 									// Give back their deposit.
-									let _ = T::Currency::unreserve(&curator, bounty.curator_deposit);
+									let err_amount = T::Currency::unreserve(&curator,
+										bounty.curator_deposit);
```

Comment:
```suggestion
									let err_amount = T::Currency::unreserve(&curator, bounty.curator_deposit);
```

Reply:
does it exceed 100? the both `curator` and `boundy` should break

Reply:
Oh, just changed it to non-breaking bc. of your former comment. Should I change it back?